templates/django/email/welcome_user.html

```diff
@@ -3,60 +3,57 @@
-    </p>
-    {% endif %}
-
-    {% if has_welcome %}
-    <p>{% blocktrans with card_number=card_number barcode_text=library.barcode_text%}Your library {{ barcode_text }} is
-        <strong>{{ card_number }}</strong>.{% endblocktrans %}
-        <br>
-        {% blocktrans with barcode_text=library.barcode_text%}Please remember this {{ barcode_text }} so you may use it
-        to access library materials available in Palace.{% endblocktrans %}
-        <br>
-        {% blocktrans with barcode_text=library.barcode_text pin_text=library.pin_text%}Use the {{ barcode_text }} and
-        the {{ pin_text }} you set when you applied to login to the Palace app.{% endblocktrans %}
-    </p>
-
-    <p>{% blocktrans with pin_text=library.pin_text%}You can go to your personal space to manage your account, manage
-        your information, or even reset your {{ pin_text }}, if needed.{% endblocktrans %}
-        <br>
-        {% blocktrans %}You can log into your personal space here: {% endblocktrans %}
-        <a href="{{ login_url }}">{{ login_url }}</a>
-    </p>
-    {% endif %}
-
-    {{ custom_bottom_text | safe }}
-
-    <p>{% blocktrans with library_name=library.name %}The {{ library_name }} team{% endblocktrans %}</p>
+
+<p>
+{% translate "Hello," %}
+</p>
+
+{% if custom_top_text %}
+<p>
+{{ custom_top_text | safe }}
+</p>
+{% endif %}
+
+<p>
+{% blocktrans with library_name=library.name %}Thank you for joining {{ library_name }}.{% endblocktrans %}
+</p>
+
+{% if has_verification %}
+<p>
+{% blocktrans %}Please verify your email address within the next 24 hours by clicking here:{% endblocktrans %} <a href="{{ verification_link }}">{{ verification_link }}</a>
+</p>
+<p>
+<em><strong>{% blocktrans %}Your account will not be activated until the link has been clicked. You may recreate a new verification email by clicking the link above again.{% endblocktrans %}</strong></em>
+</p>
+{% endif %}
+
+{% if has_welcome %}
+<p>
+{% blocktrans %}Your login details are below:{% endblocktrans %}
+</p>
+<p>
+{{ library.barcode_text }}: <strong>{{ card_number }}</strong><br>
+{{ library.pin_text }}: {% blocktranslate %}use the password you created when you signed up for a card{% endblocktranslate %}
+</p>
+<p>
+{% blocktrans %}Save this email to retain your login information. If you have forgotten your password, please visit{% endblocktrans %} <a href="{{ reset_url }}">{{ reset_url }}</a> {% blocktrans %}OR click "Forgot your password?" on the Settings - Libraries screen in the Palace app.{% endblocktrans %}
+</p>
+{%  endif %}
+
+<p>
+{% blocktrans with library_name=library.name %}You are receiving this message because someone used this email address to request a library card for {{ library_name }}. If you were not expecting this to happen, please ignore this message{% endblocktrans %}{% if has_verification %} {% translate " and do not click the verification link above." %}{% else %}.{% endif %}
```

Comment:
Hmm. This shouldn't interfere with this PR, but wondering if moving this down here might lead to some users clicking the link before they see the warning. I put a comment on the doc with the revamped text, in case anyone cares to consider it.

Reply:
That is a good point. 

The new message also doesn't link to the account page for password changes etc. Maybe I'll make a comment in the document about that as well.

I think I'll merge this as is, and we can make some more revisions depending what Carissa says in the document.